notebooks/smoothed_vision_transformers.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
Please update `notebooks/README.md` with this new notebooks

Reply:
README updated